subkey/src/main.rs

```diff
@@ -45,7 +55,7 @@ fn main() {
 				break;
 			}
 		}
-		seed = p.public().0;
+		seed = next_seed(seed);
```

Comment:
why not set seed based on some secure randomness from the OS?

Reply:
also, given that `seed` is initially set based on a randomly generated pub key, this will probably have less entropy because encoded public key points are not likely to be uniformly distributed over the key space.

Reply:
it is already seeded based on a secure random number. this just takes that number and increments through it. fair point on the pubkey point though - i'll sort that.